cli/configure/configure_test.go

```diff
@@ -18,6 +18,12 @@ func TestConfigureCli(t *testing.T) {
 
 	cmdCtx := cmdcontext.CmdCtx{}
 
+	// Remove tt file from tmp if it exists.
```

Comment:
Hmm, this is not so good because the file may have been created by some other user/service/... and it's bad practice to delete files in tests that were not created while the test was running.
I propose you a better way.
* Run the test in the newly created temporary directory in `\tmp` (`\tmp\some_random_name`)
* Remove the directory in [cleanup](https://pkg.go.dev/testing#T.Cleanup)

Reply:
Fixed, now test creates temp dir  and after cleanups it.